src/drive/web/modules/actions/index.jsx

```diff
@@ -226,16 +226,17 @@ export const move = ({ canMove, pushModal, popModal }) => {
   }
 }
 
-export const copy = ({ client, hasWriteAccess, refresh, isPublic }) => {
+export const duplicate = ({ client, hasWriteAccess, refresh, isPublic }) => {
   return {
-    icon: 'copy',
+    label: 'duplicate',
```

Comment:
Okk it works because you added label at the first place ? Still tricky but at least it works.

Reply:
@zatteo I made some change to add systematically name to be more explicit see : https://github.com/cozy/cozy-drive/pull/2809/commits/6d66c2c56b6ed33920ac6ce2ef852f01ed74af5b

Reply:
Nice! Next developer will understand this part faster than we did 😄